### ISSUE TITLE
Cute canvas background effect crash the website

### ISSUE THREAD

**Issue by faustinoaq** [amberframework/amberframework.org#103]
I just left my screen with https://amberframework.org for some hours and then I saw this:

![screenshot_20180520_060749](https://user-images.githubusercontent.com/3067335/40278448-db2b1f98-5bf6-11e8-95a9-3c16a119d9e9.png)

I reloaded the page then I tried to profile it, and I notice the animation frame is using a lot of CPU:

![screenshot_20180520_061619](https://user-images.githubusercontent.com/3067335/40278474-3749806c-5bf7-11e8-8613-d58012f027d8.png)

Also I opened my system monitor and as you can see in the capture below, the webpage is using a lot of CPU and RAM (see when I closed the webpage and the graph went down):

![screenshot_20180520_061824](https://user-images.githubusercontent.com/3067335/40278484-57f37638-5bf7-11e8-9fa5-0b9c22f29d6d.png)

I reopened the website and the CPU rocketed up again, the RAM was growing slowly and constantly.

![screenshot_20180520_062231](https://user-images.githubusercontent.com/3067335/40278485-5b2f1334-5bf7-11e8-9b8c-4f24a61014d1.png)

So, I propose to remove the canvas background, (no mobile friendly) and replace it with some CSS animation using a nice `@keyframe`, WDYT?

/cc @fridgerator @eliasjpr @amberframework/contributors

**Comment by robacarp** [comment]
> So, I propose to remove the canvas background, (no mobile friendly) and replace it with some CSS animation using a nice @keyframe, WDYT?

As much as the interactive background is cute, it doesn't really add to the site. 👍 for changing it around

**Comment by faustinoaq** [comment]
@robacarp Yeah, maybe we can use a CSS animation like this: https://codepen.io/P1N2O/pen/pyBNzX (with amber's colors of course :wink: )
